includes/elements/container.php

```diff
@@ -734,6 +758,22 @@ protected function register_background_controls() {
 				'selectors' => [
 					'{{WRAPPER}}' => '--background-transition: {{SIZE}}s;',
 				],
+				'assets' => [
```

Comment:
Do we need this code? 

I would assume that we only to conditionally load the swiper styling once.

@mugurelLupu